common/djangoapps/third_party_auth/migrations/0002_auto_20200721_1650.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.14 on 2020-07-21 16:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('third_party_auth', '0001_squashed_0026_auto_20200401_1932'),
+    ]
+
+    operations = [
```

Comment:
Is it worth running this sett of field changes by Rob Smith in case someone uses it in a particular way? I hope this default change is fine since it is only for new entries anyway but just as a safety check

Reply:
I asked George about the field values since they were specified as true in the ent-3001 ticket and he confirmed they are almost always (99% of the time) true. If they do need to be changed, these fields are still exposed in the usual Django Admin portal.

Reply:
Got it, okay thanks for following up